lib/gcloud/bigquery/data.rb

```diff
@@ -58,7 +59,7 @@ def token
 
       # The total number of rows in the complete table.
       def total
-        @gapi.total_rows
+        Service.i_from @gapi.total_rows
```

Comment:
I hate that this is even a thing. We can't just simply call `to_i` on the value because both `nil` and `""` will convert to `0`, which is not what we want. But we could use `Integer()` which is much more restrictive on how it converts types, and will raise an error if it can't convert. So I could see the implementation being:

```ruby
def total
  Integer @gapi.total_rows
rescue
  nil
end
```

My preference would be to have this all inline, but if you wanted to extract these conversion into named methods, I would prefer that they belong to something other than Service.